conf/messages

```diff
@@ -85,5 +85,11 @@ acceptanceDate.pageTitle = Was the entry acceptance date before 1 January 2021?
 acceptanceDate.heading = Was the entry acceptance date before 1 January 2021?
 acceptanceDate.error.required = Select yes if the entry acceptance date was before 1 January 2021
 
+# Supporting Documentation
+SupportDoc.pageTitle = Supporting documentation
+SupportDoc.heading = Supporting documentation
+SupportDoc.disclosure = You will need to upload files as supporting documentation for this disclosure.
+SupportDoc.fileSize = Each file must be 30MB or less, and in one of the following formats:
```

Comment:
As per previous comment the file size will need to be a parameter to this message